docs/developers_guide/organization.rst

```diff
@@ -1604,12 +1604,34 @@ an option passed in when creating this step.
   Namelist values must be of type ``str``, so use ``'{}'.format(value)`` to
   convert a numerical value to a string.
 
+
+.. _dev_step_update_namelist_options:
```

Comment:
I like that you've moved this namelist topic to live with the other namelist info.